servers/rendering_server.h

```diff
@@ -1480,6 +1480,7 @@ class RenderingServer : public Object {
 	virtual void canvas_light_occluder_set_as_sdf_collision(RID p_occluder, bool p_enable) = 0;
 	virtual void canvas_light_occluder_set_transform(RID p_occluder, const Transform2D &p_xform) = 0;
 	virtual void canvas_light_occluder_set_light_mask(RID p_occluder, int p_mask) = 0;
+	virtual void canvas_light_occluder_set_z_index(RID p_occluder, int z_index) = 0;
```

Comment:
```suggestion
	virtual void canvas_light_occluder_set_z_index(RID p_occluder, int p_z_index) = 0;
```
Prefix arguments with `p_`